src/zenml/config/docker_settings.py

```diff
@@ -122,6 +122,12 @@ class DockerSettings(BaseSettings):
             this image.
             * If a custom `dockerfile` is specified for this settings
             object, this parent image will be ignored.
+        image_tag: [Optional] Docker image tag use for generated image.
```

Comment:
```suggestion
        image_tag: Docker image tag to use for the images that get built.
```

We don't usually include `Optional` or type hints in the docstrings, all of this should already be defined in the type hints.